docs/components.md

```diff
@@ -27,7 +27,7 @@ removed prior to the 1.0 release.
 | [smartagent](../internal/receiver/smartagentreceiver)                                                                             | [span](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)                                 |                                                                                                                      |                                                                                                                              |
 | [splunk_hec](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/splunkhecreceiver)              |                                                                                                                                             |                                                                                                                      |                                                                                                                              |
 | [zipkin](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/zipkinreceiver)                     |                                                                                                                                             |                                                                                                                      |                                                                                                                              |
-
+| [journald](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/journaldreceiver)|
```

Comment:
Do you mind keeping the alphabetical ordering?  Also the receiver doc says this is in alpha.

Reply:
Thanks @rmfitzpatrick 
Moved it to alpha table.